dbt/include/global_project/macros/materializations/snapshots/helpers.sql

```diff
@@ -34,7 +34,12 @@
   {{ adapter.dispatch('snapshot_staging_table', 'dbt')(strategy, source_sql, target_relation) }}
 {% endmacro %}
 
+{% macro get_snapshot_table_column_names() %}
```

Comment:
Anything that calls this new macro will need to bump its lower bound pin on `dbt-adapters`. That's fine, just making a note of it.